clang/tools/clang-sycl-linker/ClangSYCLLinker.cpp

```diff
@@ -358,15 +483,20 @@ Error runSYCLLink(ArrayRef<std::string> Files, const ArgList &Args) {
   SmallVector<std::string> SplitModules;
   SplitModules.emplace_back(*LinkedFile);
 
-  // SPIR-V code generation step.
+  // SPIR-V code generation step and AOT compilation step.
```

Comment:
Nit: You can leave this comment as is and add a comment later about AOT compile.

Thanks